### PR TITLE
Idea document intro

### LINKED ISSUE #8: 1. Introduction & 2. Objective
1.	Introduction -> challenges, motivation
2.	Objective -> project idea(stakeholder motivate why), journal?, why is this novel, and why is this relevant to the course

•	Introduction is weak no motivation, generic stuff
•	Relevance: Only talk about which elements of information retrieval we cover and how do we do that move to method section
•	Goal: not visualize, but detect neighborhoods! Find hidden patterns in the data

### PR DESCRIPTION
**Pull request by rpjproost**
Wees kritisch, ik merk dat ik heel snel bullshit schrijf. Closes #8

### CODE REVIEW

File: Documents/idea document 2/ideadocument2.md [hunk 1]
```diff
@@ -1,6 +1,9 @@
 # Introduction
+Nowadays a lot of (social) data is gathered and available on the internet via mediums like Facebook, Twitter, Instagram and many more. These platforms are widely used by people to share information to the public. Individual tweets, status posts and photos may not be that usefull for scientific studies, but when a multitude of these tweets, posts and photos are collected, characteristics can be concluded for certain groups of individuals that for example share the same interest, like sport activities. The first challenge here, is to collect a usefull amount of data and retrieve this latent characteristics that can identify the persons interest in any kind of sport activity. The difficulty is in analyzing, categorizing and grouping data the right way, to be able to extract and come to the right conclusions. Analyzing data can give insights on how people are communicating, what they are are interested about, what motivates and drives them and just general information on what they are doing.
```
Comment (daanschipper) [inline Documents/idea document 2/ideadocument2.md]: change ize to ise (british spelling)

a lot -> many
Facebook, Twitter and Instagram. You can stop the summation here, no need to specify and many more.
usefull -> useful
the sentence "Individual tweets ... sports activities" is too long, becomes a bit confusing.
the first challenge here, -> remove ,
collect a usefull amount -> again useful, but weird sentence, when is it useful? " The first challenge is to retrieve latent characteristics from the gathered data that can..."
The difficulty is in analyzing.. -> difficulty is analysing, categorising, and ..
further in this sentence, why a comma, isn't it just and?

File: Documents/idea document 2/ideadocument2.md [hunk 2]
```diff
@@ -1,6 +1,9 @@
...
+Since data from the mentioned sources sometimes contain geographic references, it is even possible to identify and define characteristics for certain areas. Another challenge here is spatial analysis and clustering of the collected data.
```
Comment (daanschipper) [inline Documents/idea document 2/ideadocument2.md]: sometimes contain -> either contain sometimes or can contain
it is even -> do not use it, "characterisitcs can be possibly identified and defined"
~~here~~

File: Documents/idea document 2/ideadocument2.md [hunk 3]
```diff
@@ -1,6 +1,9 @@
...
 
 # Objective
+The goal of this project is to identify and characterise neighborhoods in Amsterdam based on sport activities, this requires the previously mentioned data collecting, categorization, and spatial clustering. The identified neighbourhoods can then be visualized on a map to get an overview of where different sport activities are popular, on what days of the week and on what time of the day. This can be usefull for event organizers, shopkeepers, or people that want to know where to find companions in there sport activities.
```
Comment (daanschipper) [inline Documents/idea document 2/ideadocument2.md]: change project to project name?
this -> which
are popular, on what days of the week and on what time of the day -> are popular on which time of the day and what day of the week?
refrain using this
~~usefull~~

**Review by daanschipper — CHANGES_REQUESTED** [review]
overal good text, but quite some spelling and grammar issues

File: Documents/idea document 2/ideadocument2.md [hunk 4]
```diff
@@ -1,6 +1,9 @@
 # Introduction
+Nowadays a lot of (social) data is gathered and available on the internet via mediums like Facebook, Twitter and Instagram. These platforms are widely used by people to share information to the public. Individual tweets, status posts and photos may not be that useful for scientific studies, but when a multitude of these tweets, posts and photos are collected they can be analysed together. With the collected data characteristics can be concluded for certain groups of individuals that for example share the same interest, like sport activities. The first challenge here is to collect data and retrieve the latent characteristics that can identify the persons interest in any kind of sport activity. The difficulty is in analysing, categorising and grouping data the right way to be able to extract and come to the right conclusions. Analysing data can give insights on how people are communicating, what they are are interested about, what motivates and drives them and just general information on what they are doing.
```
Comment (rubenstarmans) [inline Documents/idea document 2/ideadocument2.md]: haakjes bij social zou ik gewoon weghalen

Comment (rubenstarmans) [reply Documents/idea document 2/ideadocument2.md]: 'mediums like facebook' -> 'mediums such as'

Comment (rubenstarmans) [reply Documents/idea document 2/ideadocument2.md]: "With the collected data characteristics can be concluded for certain groups of individuals that for example share the same interest, like sport activities." Ik snap deze zin niet zo goed.. Wat bedoel je hiermee?

**Review by rubenstarmans — CHANGES_REQUESTED** [review]

**Review by rubenstarmans — APPROVED** [review]

File: Documents/idea document 2/ideadocument2.md [hunk 5]
```diff
@@ -1,6 +1,9 @@
 # Introduction
+Nowadays a lot of social data is gathered and available on the internet via mediums such as Facebook, Twitter and Instagram. These platforms are widely used by people to share information to the public. Individual tweets, status posts and photos may not be that useful for scientific studies, but when a multitude of these tweets, posts and photos are collected they can be analysed together. With the collected data characteristics can be extracted for different groups of individuals that for example share the same interest, like sport activities. The first challenge here is to collect data and retrieve the latent characteristics that can identify the persons interest in any kind of sport activity. The difficulty is in analysing, categorising and grouping data the right way to be able to extract and come to the right conclusions. Analysing data can give insights on how people are communicating, what they are are interested about, what motivates and drives them and just general information on what they are doing.
```
Comment (daanschipper) [inline Documents/idea document 2/ideadocument2.md]: a lot of -> "social data is gathered in large quantities"
I do not like the use of a lot a lot

**Review by daanschipper — CHANGES_REQUESTED** [review]

**Review by daanschipper — APPROVED** [review]

**Pull request merged**